.gitattributes

```diff
@@ -0,0 +1 @@
+alternate_formats/* -diff
```

Comment:
also @konklone, what are your thoughts on the json formatting being all one line.  Saves space (a few MB on the historical file...), and similar to the .gitattributes thing i was trying it makes looking at diffs on github a little easier (IMO, but obviously not looking at diffs within the json itself.  But in theory we can trust the diff matches if the build passes anyways)